src/deploy/osp_deployer/profiles/xsp.json

```diff
@@ -4,13 +4,26 @@
    "associated_settings": [
       {
          "Dell NFV Settings": {
-            "hpg_enable": "",
-            "numa_enable": ""
+            "ovs_dpdk_enable": {
```

Comment:
Can you add the sections for ovs_dpdk_role_list and ovs_dpdk_policy to this section?

Reply:
Sure, my fault! It's added now. I also changed the order of the options in the csp.json file to make it consistent.